### PR TITLE
BrijeshG_Exam_1_17Minutes

### PR DESCRIPTION
**Pull request by BrijeshGOct2020**
image

### CODE REVIEW

File: src/brijesh/codingexams/Exam1.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+import javax.xml.ws.Dispatch;
+
+public class Exam1 {
+
+	String getMaxLengthName(String[] arr) {
+		int max=arr[0].length();
+		String maxString=arr[0];
+		for(int i=1; i<arr.length;i++) {
+			if(arr[i].length()>max) {
```
Comment (KrishnaKTechnocredits) [inline src/brijesh/codingexams/Exam1.java:14]: max variable can be removed. 
if(arr[i].length() > maString.length())

File: src/brijesh/codingexams/Exam1.java [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+				maxString=arr[i];
+			}
+		}
+		System.out.println("String which has maximum length\n"+maxString+" ----> "+max);
+		return maxString;
+	}
+
+	void display(String name) {
+		System.out.println("Even characters of name "+name);
+		for(int i=0; i<name.length();i++) {
```
Comment (KrishnaKTechnocredits) [inline src/brijesh/codingexams/Exam1.java:25]: use index instead of i please.

**Review by KrishnaKTechnocredits — APPROVED** [review]
Code looks good to me Brijesh.

File: src/brijesh/codingexams/Exam1.java (hunk 1 above)
Comment (BrijeshGOct2020) [reply src/brijesh/codingexams/Exam1.java:14]: Yes Thanks Got it.

**Pull request merged**